templates/test/overview.html.ep

```diff
@@ -89,6 +89,12 @@
                     <input type="text" class="form-control" name="arch" placeholder="any" id="filter-arch">
                 </div>
                 <div class="form-group">
+                    <strong>Failed modules</strong>
+                    <input name="failed_modules" type="text" id="filter-failed_modules" placeholder="shutdown,install_and_reboot">
```

Comment:
I would not write specific examples in the placeholder but a generic description, e.g. as in L89 "any". How about "failed modules, comma separated, e.g. mod1,mod2"? or just the examples as you gave them but with a prepended 'e.g.'